pkg/utils/utils.go

```diff
@@ -0,0 +1,10 @@
+package utils
+
+import (
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+func Title() cases.Caser {
+	return cases.Title(language.English, cases.NoLower)
+}
```

Comment:
```suggestion
var Caser = cases.Title(language.English, cases.NoLower)
```
then just call `helper.Caser.String(...)`